### DISCUSSION TITLE
Default prBodyNotes for replacements

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#15004] [Suggest an Idea]
All replacements have an extra `prBodyNotes`:
```
'This is a special PR that replaces `{{{depNameSanitized}}}` with the community suggested minimal stable replacement version.'
```
 It appears that that cannot be disabled, even though the text may be confusing or downright incorrect:
- Replacements can have the same `newName` as `depName`. If so, the project in `depNameSanitized` is not replaced.
- Renovate (luckily) does not enforce that the replacement renovates the dependency to the "minimal stable replacement version".

Could we address the textual issues, and/or allow users to disable the extra text? Should we maybe move the property from `lib/config/options/index.ts` into its own preset, and have that preset included in `config:base` (or its sub-presets), so that they can be ignored using `ignorePresets`?

**Comment by rarkins** [comment]
Do you have an example of where depName is unchanged?

**Reply by astellingwerf** [reply]
In [!14962](https://github.com/astellingwerf/renovate-issue-14962/blob/master/renovate.json5), where I use replacements to switch between Docker compatibilities.

Or a more speaking example:
```json
    {
      "matchDatasources": ["docker"],
      "matchPackageNames": ["artifactory.domain.tld/node"],
      "replacementName": "artifactory.domain.tld/node",
      "matchCurrentVersion": "/^16\\.13\\.0\\.202111\\.342357-alpine3\\.14$/",
      "replacementVersion": "16.13.2.202201.393988-alpine3.15",
      "prBodyNotes": "This PR helps you to jump over to the Alpine 3.15 track, since that's what the latest hardened versions of Node.js 16 are shipped with.",
      "branchTopic": "{{{depNameSanitized}}}-alpine3.15-replacement"
    },
```

**Reply by rarkins** [reply]
I guess one problem is that if we let people define their own replacements then the words "the community" are wrong to begin with?

**Reply by astellingwerf** [reply]
Yeah, well, I decided not to object to that one (because "we're all a community" I guess), but indeed, one might take offence at that one as well.
